pages/00-Home.md

```diff
@@ -70,18 +24,31 @@ The hydrus system is built over the following standards and tools:
 
 Other than this, there are also various Python packages that hydrus uses. A list of these packages can be found in the [requirements.txt](https://github.com/HTTP-APIs/hydrus/blob/master/requirements.txt) file.
 
+## Hydra Smart Clients
+
+**Hydra smart client** (`hydra-python-agent`) is a Python implementation of a client based on the Hydra Community Group specifications and reference implementation [Heracles.ts](https://github.com/HydraCG/Heracles.ts). It can navigate a network of hydrus servers autonomously by reading their API documentation. It is a generic client because it can query and retrieve data from any, Hydra-aware or not, server (such as hydrus or any other Web server).
```

Comment:
ATM the client can only connect with a single hydrus server, how can it navigate through a network of hydrus servers?

Reply:
`It is a generic client because it can query and retrieve data from any, Hydra-aware or not, server (such as hydrus or any other Web server).`
Only works with the hydrus server or hydra based if we consider future developments